third_party/cpgfunctionEP-0.3.2/include/cpgfunction/boreholes.h

```diff
@@ -0,0 +1,96 @@
+//
```

Comment:
Header file for boreholes. All things related to borehole properties and positions are in this module.